docs/website/src/site/scripts/tabs.js

```diff
@@ -8,7 +8,7 @@ for (let i = 0; i < tabs?.length; i++) {
     }
 
     const current = tabsContainer.querySelectorAll('.active');
```

Comment:
Why do `querySelectorAll` if you're only going to use the first one? Simply do `querySelector`?

Reply:
right :)

Reply:
this is an old version of tabs, I forgot to remove it. The right file is in `ts/scripts/tabs.ts` file